game/domain/params/choose_widget_slot.lua

```diff
@@ -11,7 +11,7 @@ function PARAM.isValid(sector, actor, parameter, value)
   if not actor:getBody():hasWidgetAt(value) then
     return false
   end
-  return true
+  return not not actor:getBody():getWidget(value):getWidgetActivation()
```

Comment:
This only works when we choose an widget to activate. It won't work for ablities that require a target widget to destroy, for example.

Reply:
yes but for now it makes the menu not close when i select an invalid widget

Reply:
Ok, I'll make an issue for when we have abilities that target widgets